content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -129,6 +129,24 @@ cloudprovider_gce_api_request_duration_seconds { request = "detach_disk"}
 cloudprovider_gce_api_request_duration_seconds { request = "list_disk"}
 ```
 
+
+### kube-scheduler metrics
+
+{{< feature-state for_k8s_version="v1.20" state="alpha" >}}
+
+The scheduler exposes optional metrics that reports the requested resources and the desired limits of all running
+pods. These metrics can be used to build capacity planning dashboards, assess current or historical scheduling limits,
+quickly identify workloads that cannot schedule due to lack of resources, and compare actual usage to the pod's request.
+Every pod reports one metrics series for every non-zero resource request or limit. The pod namespace, name, scheduled
+node, priority, and assigned scheduler are reported as labels. Once a pod reaches completion (has a `restartPolicy` of
```

Comment:
Is this accurate:
```suggestion
The kube-scheduler identifies the resource
[requests and limits](/docs/concepts/configuration/manage-resources-containers/)
configured for each Pod; when either a request or limit is non-zero, the kube-scheduler
reports a metrics timeseries. The time series is labelled by:

- namespace
- pod name
- the node where the pod is scheduled
  (the time series only published once the pod is assigned to a node)
- priority
- the assigned scheduler for that pod
- the name of the resource (for example, `cpu`)

Once a pod reaches completion (has a `restartPolicy` of
```
?

Reply:
Two fixes:

* The time series is reported even if the pod is not scheduled
* The resource unit is reported as a label when known (for resources defined in the Kubernetes model)